### PR TITLE
Spm implementation

### PR DESCRIPTION
**Pull request by raypinto**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import PackageDescription
+
+let package = Package(
+    name: "Prototyper",
+    platforms: [
+        .iOS(.v13),
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
```
Comment (PSchmiedmayer) [inline Package.swift]: Feel free to remove the boilerplate comments from Package.swift

File: Tests/PrototyperTests/PrototyperTests.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import XCTest
+@testable import Prototyper
+
+final class PrototyperTests: XCTestCase {
+    func testExample() {
+        // This is an example of a functional test case.
```
Comment (PSchmiedmayer) [inline Tests/PrototyperTests/PrototyperTests.swift]: Feel free to remove boilerplate comments and commented code

File: Sources/Prototyper/Classes/Extensions/UIImage+OverlapImages.swift [hunk 3]
```diff
@@ -6,6 +6,9 @@
 //
 
 import Foundation
+#if canImport(UIKit)
+import UIKit
```
Comment (PSchmiedmayer) [inline Sources/Prototyper/Classes/Extensions/UIImage+OverlapImages.swift:9]: As far as I can see the Swift Package seems to build fine without the `#if canImport(UIKit)` statements across the code. It builds successfully using Xcode when selecting an iOS target. I would suggest to remove the if statements.

**Review by PSchmiedmayer — CHANGES_REQUESTED** [review]
I would suggest to remove the following files:
- Prototyper.podspec
- All CocoaPods files in the example project

In addition I would suggest to add the Prototyper dependency in the Xcode Project using the SPM. I would suggest to use `pod deintegrate` (https://guides.cocoapods.org/terminal/commands.html#pod_deintegrate) to remove all remaining pod files and settings in the Xcode project. I would suggest to add Prototyper as an SPM dependency without a version constraint and just always use the latest version from master, this might make it easier for us to test the functionality of the package.

**Comment by PSchmiedmayer** [comment]
Good job with this PR, the structure looks good. When we have removed all Cocoapods occurrences we can merge this branch 👍

**Comment by raypinto** [comment]
> Good job with this PR, the structure looks good. When we have removed all Cocoapods occurrences we can merge this branch 👍

**Review by PSchmiedmayer — CHANGES_REQUESTED** [review]
I would suggest to move the `fastlane` folder in the example folder as it is related to the example app and not the swift package itself 👍

**Review by PSchmiedmayer — APPROVED** [review]

**Pull request merged**